negotiation/src/main/java/it/eng/negotiation/rest/api/ContractNegotiationAPIController.java

```diff
@@ -0,0 +1,65 @@
+package it.eng.negotiation.rest.api;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import com.fasterxml.jackson.databind.JsonNode;
+
+import it.eng.negotiation.model.ContractAgreementVerificationMessage;
+import it.eng.negotiation.service.ContractNegotiationAPIService;
+import it.eng.negotiation.service.ContractNegotiationEventHandlerService;
+import it.eng.tools.event.contractnegotiation.ContractNegotiationOfferResponse;
+import it.eng.tools.model.DSpaceConstants;
+import lombok.extern.slf4j.Slf4j;
+
+@RestController
+@RequestMapping(consumes = MediaType.APPLICATION_JSON_VALUE, produces = MediaType.APPLICATION_JSON_VALUE, path = "/api/negotiations")
+@Slf4j
+public class ContractNegotiationAPIController {
+	
+	@Autowired
```

Comment:
Same question as above.

Reply:
All Autowired are converted to constructor injection, and I will stick (like I did before) to this one, and not believe everything I read on Stackoverflow